context/test-utils/loaders/ymlLoader.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Are we doing something odd that this disable is needed here and elsewhere? Perhaps turn if off in test dirs?

Reply:
eslint doesn't recognize our webpack aliases/jest paths. I'll create an issue to think about how to fix this.